doccano_client/client.py

```diff
@@ -357,7 +357,7 @@ def create_project(
         allow_overlapping: bool = False,
         grapheme_mode: bool = False,
         use_relation: bool = False,
-        tags: Optional[List[str]] = None,
+        tags: Optional[List[Dict[str, str]]] = None,
```

Comment:
In create/update methods I set this to `Optional[List[Dict[str, str]]]` b/c as far as I can tell the accepted format for a tag in these calls is simply `[{"text", "<tag_name>"}, ...]` and am unsure if it would be accepted by the API to pass any keys aside from `"text"` (i.e. `id` or `project`) in the `tags` dictionaries. Happy to change this to `Optional[List[Dict[str, Any]]]` if it's felt this would be more appropriate